src/specific/s_filesystem.h

```diff
@@ -1,4 +1,7 @@
 #pragma once
 
+#include <stdbool.h>
```

Comment:
This isn't needed now

Reply:
Oh yes. I also added `#pragma once` since it was missing for `s_filesystem.h`.